### PR TITLE
ATO-160: Update stub documentation

### PR DESCRIPTION
**Pull request by CarlyG55**
## What?

Update the documentation for the stub

## Why?

To make it easier to use and develop on

### CODE REVIEW

**Review by mrwilson — DISMISSED** [review]

**Review by ethanmills — DISMISSED** [review]

File: README.MD [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
+It loads a screen where you can set various parameters (see [here]("https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/")), and make calls to the `/authorize` endpoint using these settings.
+Once you have gone through the authentication/identity journey, you will be redirected to the stub, with a page
+displaying the user info returned by the One Login service.
 
 ## Local Setup
 You'll need to set up environment variables to run locally. Make a copy of the `.env.template` file, and rename it `.env`.
+
 The stub is currently set up to connect to the build environment (as sandpit blocks requests from localhost). Once you 
 have a copy of `.env`, you'll need to go into the `build-client-registry` table in DynamoDB, and find the client with ClientName `di-auth-stub-relying-party-build`.
 Update the value of `CLIENT_ID` to the ClientID of this client.
```
Comment (hannah-leong) [inline README.MD:14]: For the Doc App stub, does the `CLIENT_ID` need to be set to the `di-auth-stub-relying-party-build-app` value?
```suggestion
The stub is currently set up to connect to the build environment (as sandpit blocks requests from localhost). Once you 
have a copy of `.env`, you'll need to go into the `build-client-registry` table in DynamoDB, and find the client with ClientName `di-auth-stub-relying-party-build` or `di-auth-stub-relying-party-build-app` for Doc App.
Update the value of `CLIENT_ID` to the ClientID of this client.
```

File: README.MD [hunk 2]
```diff
@@ -18,8 +27,31 @@ Note that we have renamed the `sandpit` environment to `dev` to align with secur
...
 IPV have a tool that manages secrets in the stubs AWS account. If you need to add any further Secrets Manager secrets, 
 you should add the name of the secret to the externally managed list [here](https://github.com/govuk-one-login/ipv-stubs-common-infra/blob/main/utils/config-mgmt/app/configs/stubs.build.params.yaml) for non prod,
-or [here](https://github.com/govuk-one-login/ipv-stubs-common-infra/blob/main/utils/config-mgmt/app/configs/stubs.production.params.yaml) for prod.
\ No newline at end of file
+or [here](https://github.com/govuk-one-login/ipv-stubs-common-infra/blob/main/utils/config-mgmt/app/configs/stubs.production.params.yaml) for prod.
+
+## Deploying a new copy of the stub
+
+At the moment, to deploy a new copy of the stub you should duplicate the resources in the template (except the lambda authorizer,
+KMS key and Fargate cluster resources). You can see any example of how this is done by looking at the Performance Testing resources.
```
Comment (hannah-leong) [inline README.MD:43]: Might be worth linking the Performance Testing resources here?

**Review by hannah-leong — DISMISSED** [review]

**Comment by hannah-leong** [comment]
LGTM, feel free to resolve my comments if they aren't needed

File: README.MD (hunk 2 above)
Comment (CarlyG55) [reply README.MD]: Have updated to clarify that they're in the template - is that what you were looking for? 🙂

Comment (hannah-leong) [reply README.MD]: Ah I just read it wrong, thought it was generic resources (i.e., documentation somewhere else not terraform resource 🤦‍♀️ )

**Review by hannah-leong — DISMISSED** [review]

File: README.MD [hunk 3]
```diff
@@ -18,8 +27,32 @@ Note that we have renamed the `sandpit` environment to `dev` to align with secur
...
\ No newline at end of file
+or [here](https://github.com/govuk-one-login/ipv-stubs-common-infra/blob/main/utils/config-mgmt/app/configs/stubs.production.params.yaml) for prod.
+
+## Deploying a new copy of the stub
+
+At the moment, to deploy a new copy of the stub you should duplicate the resources in the template (except the lambda authorizer,
+KMS key and Fargate cluster resources). You can see any example of how this is done by looking at the Performance Testing resources
+in `template.yaml` in the root directory.
+
+If you would like to deploy a new copy of the stub, you'll need to add it to the client registry for the relevant environment.
```
Comment (isaac-GDS) [inline README.MD:46]: Not sure - but would you also need to re-deploy the manual stacks resources ?

Comment (CarlyG55) [reply README.MD:46]: Yes, good point, I'll add that 🙂

Comment (CarlyG55) [reply README.MD:46]: Updated it - let me know if you think that's clear enough 🙂

**Review by ethanmills — APPROVED** [review]

File: README.MD (hunk 3 above)
Comment (isaac-GDS) [reply README.MD:46]: Looks good

**Review by isaac-GDS — APPROVED** [review]

**Pull request merged**